terraform/module/variables.tf

```diff
@@ -6,6 +6,12 @@ variable "lambda_function_prefix" {
   default = "lambda_power_tuning"
 }
 
+variable "role_path_prefix" {
+  default     = null
```

Comment:
If we end up using this as a string prefix, the default value could be an empty string (`""` instead of `null`). This could make the prefixing logic slightly simpler :)